ydb/core/tx/datashard/datashard_user_table.h

```diff
@@ -263,10 +262,12 @@ struct TUserTable : public TThrRefBase {
         TTableIndex() = default;
 
         TTableIndex(const NKikimrSchemeOp::TIndexDescription& indexDesc, const TMap<ui32, TUserColumn>& columns)
-            : Name(indexDesc.GetName())
-            , Type(indexDesc.GetType())
+            : Type(indexDesc.GetType())
             , State(indexDesc.GetState())
         {
+            if (Type != EType::EIndexTypeGlobalAsync) {
+                return;
+            }
```

Comment:
We don't need fill these vectors for other types of indexes.
Motivated by this review comment https://github.com/ydb-platform/ydb/pull/4967#discussion_r1668881097

Reply:
Be aware of CDC index usage!
https://github.com/ydb-platform/ydb/blob/8c6ee313319e101151d5d233811a266a805bdb03/ydb/core/tx/datashard/change_record_cdc_serializer.cpp#L373

Reply:
This place also was adjusted